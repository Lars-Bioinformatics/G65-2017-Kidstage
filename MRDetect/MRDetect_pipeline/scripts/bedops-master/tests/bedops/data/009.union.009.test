chr1	20	100
chr1	30	50
chr2	5	8
chr2	50	80
