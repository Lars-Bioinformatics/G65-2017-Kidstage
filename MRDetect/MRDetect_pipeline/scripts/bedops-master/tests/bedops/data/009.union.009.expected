chr1	15	92
chr1	25	42
chr2	45	72
